.github/workflows/cypress.yml

```diff
@@ -0,0 +1,52 @@
+name: Cypress
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Environment
+        uses: actions/checkout@v2
+        with:
+          repository: orangehrm/orangehrm-os-dev-environment
+
+      - name: Checkout OrangeHRM
+        uses: actions/checkout@v2
+        with:
+          path: html
+
+      - name: Configure Docker Environment
+        run: |
+          pwd
+          ls -la
+          ls html
+          cp .env.dist .env
+          sed -i "s~LOCAL_SRC=/your/src/path~LOCAL_SRC=$PWD/html~g" .env
+          echo "127.0.0.1 php56 php70 php71 php72 php73 php74 php80" | sudo tee -a /etc/hosts
+          docker-compose -f docker-compose.yml -f docker-compose-build.yml build nginx php-8.0
+          docker-compose up -d php-8.0 mysql55
```

Comment:
we may able to cache the docker images .. otherwise it costly operation 
https://evilmartians.com/chronicles/build-images-on-github-actions-with-docker-layer-caching

Reply:
Docker image cache and yarn cache handled. Above approach not worked
Using https://github.com/actions/cache, https://docs.docker.com/engine/reference/commandline/save/, https://docs.docker.com/engine/reference/commandline/load/